sample/README.md

```diff
@@ -0,0 +1,48 @@
+# Sample
+
+The Sample describes several demos of quickly spinning up different Prebid Server instances with various preset configurations. These samples are intended for audiences with little knowledge about Prebid Server and plan to play around with it locally and see how it works.
+
+# Installation
+
+In the Sample, we use `docker` and `docker-compose` to instantiate examples; with docker providing a unified setup and interface,  you can spin up a demo server instance locally with only one command without knowing all the complexities.
+The docker image used in `docker-compose.yml` is the `Dockerfile` residing in the root level of the repository. 
+
+## Option 1 - Standard Docker Engine
+Install `docker` and `docker-compose` via the [official docker page](https://docs.docker.com/compose/install/#scenario-one-install-docker-desktop). If you cannot use the official docker engine due to restrictions of its license, see the option below about using Podman instead of Docker. 
+
+## Option 2 - Podman
+From MacOS, you can use [podman](https://podman.io/) with these additional steps:
+
+```sh
+$ brew install podman docker-compose
+$ podman machine init
+$ podman machine set --rootful
+$ podman machine start
+$ cd sample
+$ docker-compose up <number>_<name>
+```
+
+# Examples
+
+## Common File & Structures
+All required files for each example are stored in a folder that follows the name pattern <number>_<name>. The `<number>` suggests its order and `<name`>` describes its title.
+
+The following files will be present for every example and are exclusively catered to that example.
+1. `app.yaml` - the prebid server app config.
```

Comment:
Can you please specify here, that no other config files must be present in the Prebid server root, like `pbs.json` or `pbs.yaml`? 
For some reason while running `docker-compose up` my project root `pbs.json` was used for the main config.

Reply:
hmm, it should not be, since the docker-compose file uses the volume mount to overwrite the config, let me check it again, there might because the ordering of  viper config parsing process

Reply:
@VeronikaSolovei9 looked deep into it, yes, the viper looks for `pbs.yaml` so it overwrites the `app.yaml`.

i have updated the `Steps` to include that we should not have other config files